halo2-snark-aggregator-api/src/systems/halo2/verify.rs

```diff
@@ -655,50 +657,36 @@ fn evaluate_multiopen_proof<
     schip: &A::ScalarChip,
     pchip: &A,
     proof: MultiOpenProof<A>,
-    //params: &ParamsVerifier<E>,  only for debugging purpose
+    _params: &ParamsVerifierKZG<E>,
 ) -> Result<(A::AssignedPoint, A::AssignedPoint), A::Error> {
     let one = schip.assign_one(ctx)?;
 
-    println!("debug context before evaluate multiopen proof: {}", ctx);
-    let mut points = Vec::new();
-    let (left_s, left_e, mut points_wx) = proof.w_x.eval::<_, A>(ctx, schip, pchip, &one)?;
-    let (right_s, right_e, mut points_wg) = proof.w_g.eval::<_, A>(ctx, schip, pchip, &one)?;
-    points.append(&mut points_wx);
-    points.append(&mut points_wg);
-    print_points_profiling(&points);
```

Comment:
Previously we printed ecmul of each circuit here. Now this feature disappeared. Can it be recovered？

Reply:
The print points profiling function